x-pack/plugins/security_solution/common/api/detection_engine/rule_management/coverage_overview/coverage_overview_route.test.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { exactCheck, foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
+import {
+  CoverageOverviewRequestBody,
+  CoverageOverviewRuleActivity,
+  CoverageOverviewRuleSource,
+} from './coverage_overview_route';
```

Comment:
Obviously not part of this PR, but these types need to be migrated to Zod. It makes testing these request schemas SO much easier and understandable.

Reply:
I agree, I saw the validation tests Dmitrii wrote with zod while writing these and thought the same. In a later PR these types definitely need to be switched over.